lib/tasks/users/get_an_identity_syncing.rake

```diff
@@ -44,4 +44,46 @@ namespace :users do
       end
     end
   end
+
+  namespace :get_an_identity_data_sync do
```

Comment:
Will this get used that often? as its only for one user right?

Reply:
Also think description could be updated

Reply:
I added it mainly as a backup, in case we get a few individual failures or we get asked to push through changes for a specific user we can use this to sort it immediately rather than needing to use the console at all. Might not be used often but it'll be really helpful if we ever need it.

Reply:
ok no worries yeah i got confused and thought youd just use the other one sending an array with one user